PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -70,13 +70,17 @@ public abstract class PayPalRequest implements Parcelable {
     private String merchantAccountId;
     private String riskCorrelationId;
     private final ArrayList<PayPalLineItem> lineItems;
+    private final boolean hasUserLocationConsent;
 
     /**
      * Constructs a request for PayPal Checkout and Vault flows.
+     *
+     * TODO: add javadoc for hasUserLocationConsent param
      */
-    public PayPalRequest() {
```

Comment:
This should be deprecated not removed I think

Reply:
Likely should be non-public in v5 since I don't think merchants ever use it but still _technically_ a breaking change

Reply:
That was my thought in removing the empty constructor. I'll add the deprecated empty constructor back.

Do we always handle breaking changes for flows that are potentially misuse of the SDK?

Reply:
Historically I think we have still treated misuse as breaking changes, and added `NEXT_MAJOR_VERSION` notes to resolve, but open to the rest of the teams thoughts on that